connector/countconnector/count.go

```diff
@@ -0,0 +1,214 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package countconnector // import "go.opentelemetry.io/collector/connector/countconnector"
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/internal/sharedcomponent"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+const (
+	typeStr = "count"
+)
+
+type LogsConfig struct {
+	MinSeverity string              `mapstructure:"min_severity"`
+	minSevNum   plog.SeverityNumber // internal use
+}
+
+func (c *LogsConfig) setup() {
+	switch strings.ToLower(c.MinSeverity) {
+	case "fatal":
+		c.minSevNum = plog.SeverityNumberFatal
+	case "error":
+		c.minSevNum = plog.SeverityNumberError
+	case "warn":
+		c.minSevNum = plog.SeverityNumberWarn
+	case "info":
+		c.minSevNum = plog.SeverityNumberInfo
+	case "debug":
+		c.minSevNum = plog.SeverityNumberDebug
+	default:
+		c.minSevNum = plog.SeverityNumberUnspecified
+	}
+}
+
+type Config struct {
+	config.ConnectorSettings `mapstructure:",squash"`
+	LogsConfig               `mapstructure:"logs"`
+}
+
+var _ config.Connector = (*Config)(nil)
+
+// NewFactory returns a ConnectorFactory.
+func NewFactory() component.ConnectorFactory {
```

Comment:
Cannot find `component.ConnectorFactory`, did you miss it?

Reply:
It's in `component/connector.go`